### PR TITLE
Feature #r103.1 create edit institution of education page

### PR DESCRIPTION
**Pull request by andrii-sharun**
dev
## CLICKUP
https://app.clickup.com/t/fv7fjj

## Code reviewers
@Vasyl3611 
@KostiantynKukh 
@Ormared 

## Summary of issue
The representative of the educational institution should be able to change the information about the educational institution

## Summary of change
A page has been created where a representative of the educational institution has the opportunity to edit the data.

## Testing approach
React Testing Library

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/superAdmin/addInstitutionOfEducationForm/map/index.tsx [hunk 1]
```diff
@@ -13,7 +13,8 @@ const GetIcon = () => {
 };
 
 const UniversityMap = (props: any) => {
-  const [position, setPosition] = useState<[number, number]>([0, 0]);
+  const curent = props.curentPosition ? props.curentPosition : [0, 0];
```
Comment (Ormared) [inline src/components/superAdmin/addInstitutionOfEducationForm/map/index.tsx]: Простіше це записувати як:
`let current =props.currentPosition || [0,0]`

Comment (andrii-sharun) [reply src/components/superAdmin/addInstitutionOfEducationForm/map/index.tsx]: Дякую.Візьму до уваги!

File: src/pages/institutionOfEducationAdminPage/institutionOfEducationInfo/index.tsx [hunk 2]
```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import styles from './institutionOfEducationInfo.module.scss';
-import { Link } from 'react-router-dom';
+import { Link, useRouteMatch } from 'react-router-dom';
 
 const InstitutionOfEducationInfo = () => {
-  const { name, abbreviation, site, address, phone, email, description } = {
+  const { path, url } = useRouteMatch();
+  console.log(path);
+  console.log(url);
```
Comment (Ormared) [inline src/pages/institutionOfEducationAdminPage/institutionOfEducationInfo/index.tsx]: delete logs

Comment (andrii-sharun) [reply src/pages/institutionOfEducationAdminPage/institutionOfEducationInfo/index.tsx]: Deleted!

**Review by KostiantynKukh — APPROVED** [review]

**Review by Vasyl3611 — APPROVED** [review]

**Review by Ormared — APPROVED** [review]

**Pull request merged**